codespell_lib/data/dictionary.txt

```diff
@@ -1244,18 +1244,17 @@ analitic->analytic
 analitical->analytical
 analitically->analytically
 analiticaly->analytically
-analize->analyze
-analized->analyzed
-analizer->analyzer
-analizes->analyzes
-analizing->analyzing
+analize->analyze, analyse,
+analized->analyzed, analysed,
+analizer->analyzer, analyser,
+analizes->analyzes, analyses,
+analizing->analyzing, analysing,
 analogeous->analogous
 analoguous->analogous
 analoguously->analogously
 analyis->analysis
-analysator->analyzer
-analysers->analyzers
-analysies->analyzes
+analysator->analyzer, analyser,
+analysies->analyzes, analyses,
```

Comment:
Suggested: `analysies->analyses, analysis,`

Reply:
I'd agree with @JMcKiern 's suggestion of the zes variant too, resolving.